### PR TITLE
feat: create mvp dapp sample

### PR DESCRIPTION
**Pull request by PainterPuppets**
create mvp dapp by `kuai init`

### CODE REVIEW

**Review by Daryl-L — APPROVED** [review]

File: packages/samples/mvp-dapp/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "arrowParens": "always"
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/.prettierrc:5]: A point was neglected before that it's better to keep the rule consistent: https://github.com/ckb-js/kuai/blob/develop/.prettierrc#L6

File: packages/samples/mvp-dapp/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "kuai-mvp-dapp",
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@ckb-js/kuai-core": "0.0.1",
+    "@ckb-js/kuai-io": "0.0.1",
+    "@ckb-js/kuai-models": "0.0.1",
+    "koa": "^2.14.1",
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/package.json:10]: Dependencies should be pinned

**Comment by Keith-CY** [comment]
It seems that codes are quite different from what we've drafted in https://github.com/Magickbase/kuai-mvp-server/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R8

**Comment by Keith-CY** [comment]
Skeleton code in `controller` was simplified

**Comment by yanguoyu** [comment]
After finishing MVP-DApp, we need to perfect the init project template.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**